### PR TITLE
Ruby remove booleans from numbers

### PR DESCRIPTION
**Pull request by dvik1950**
From https://github.com/exercism/v3/issues/2046:

- Make `booleans` a prerequisite of `numbers` -> Done
- Remove Boolean-content from `numbers`, ensure the content appears in `booleans` -> Done (at least I think so). The content I removed here is covered in the [introduction.md](https://github.com/exercism/v3/blob/master/languages/ruby/exercises/concept/booleans/.docs/introduction.md) of the booleans exercise.

Please let me know if I missed something, thanks.

### CODE REVIEW

File: languages/ruby/exercises/concept/numbers/.meta/design.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ The Concepts this exercise unlocks are:
 This exercise's prerequisite Concepts are:
 
 - `basics`: know how to define methods.
+- `booleans`: know about `true`/`false` and truthy/falsy values.
```
Comment (kotp) [inline languages/ruby/exercises/concept/numbers/.meta/design.md:30]:
```suggestion
- `booleans`: know about `true`/`false` and truthy/falsey values.
```

**Comment by ErikSchierboom** [comment]
> Not sure why the formatter didn't format the readme, but that is the failing check. If you get stuck I am happy to format it locally, or if you have node installed and the repo locally, you can use npx prettier on the file.

I've just enabled an auto-format run by commenting with the text "/format", which should fix the issue.

> For the pre-requisites, I don't think it needs to also list basics. It isn't wrong (AFAIK, because I did the same think in many instances on the elixir track) but it doesn't seem to be the trend to list transitive ancestors in this track.

This is true, although that rule is mostly about the prerequisites in the track's `config.json` file. I don't mind the `basics` prerequisite in the `design.md` document, as long as it isn't there in the track's `config.json` file.

File: languages/ruby/exercises/concept/numbers/.meta/design.md (hunk 1 above)
Comment (ErikSchierboom) [inline languages/ruby/exercises/concept/numbers/.meta/design.md:30]: @dvik1950 Could you also update the `languages/ruby/config.json` file to have this new prerequisite? It is the `languages/ruby/config.json` that the v3 website will use to get the concept and prerequisite information for exercises. The `design.md` document is just there as documentation.

Comment (dvik1950) [reply languages/ruby/exercises/concept/numbers/.meta/design.md]: Sorry, no idea how I missed that one

Comment (dvik1950) [reply languages/ruby/exercises/concept/numbers/.meta/design.md]: I searched the V3 repo and both `falsy` and `falsey` seem to appear almost equally. We should probably agree on on in the repo

**Comment by dvik1950** [comment]
> I've just enabled an auto-format run by commenting with the text "/format", which should fix the issue.

I think I [run the comment](https://github.com/exercism/v3/pull/2059#issuecomment-662056975) already and it did not seem to fix the issue 🤔 I run prettier locally and now it's ok

**Pull request merged**